### ISSUE TITLE
DevTools: Failed to open the manifest.json file of plugin

### ISSUE THREAD

**Issue by MahdeenSky** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#75]
[general] DevTools: Failed to open the manifest.json file of plugin Failed to open the file.

I'm not sure why I am getting the error above, I can access the path fine, and edit it using vim. Why can't it access this json?

**Comment by AbdullahAlfaraj** [comment]
@MahdeenSky it could be a permission issue. 
also did you check enable development mode in photoshop?
![image](https://user-images.githubusercontent.com/7842232/215420251-d20a1284-ed6f-4ce9-b9f6-fdb38b994bd8.png)

**Comment by MahdeenSky** [comment]
Yes i did

![Screen Shot 2023-01-30 at 4 03 41 AM](https://user-images.githubusercontent.com/49484385/215433100-f64ca00a-a2c0-4519-93c8-6d6c38cf5f6f.png)

**Comment by MahdeenSky** [comment]
Also gave the Adobe Developer UXP Tool, full disk access perms. Still same error.

**Comment by AbdullahAlfaraj** [comment]
@MahdeenSky what path do you have the plugin installed at? 
some people had issues when placing the plugin in the root directory like C:
at any rate we are working on a ccx package, so we could stop relaying on uxp. 
expected soon.

**Comment by latentwiki** [comment]
I had this problem when the plugin files were stored on a separate volume, different than where Photoshop is installed.  Try copying/re-cloning to the same volume (where PS is) and re-add the plugin from that new location.  I encountered this on Mac OSX (latest) and I also have Bitdefender fully active.

I've found a workaround as mentioned, but I will try looking into app log files to pinpoint why this happens if I can still reproduce it.  I'll reach out or see if I can open a PR if I find anything.

I assume that it's some permission bug with the Adobe app as a root cause... But that's just my initial guess. 😋
